Cloud Application Manager/Monitoring/CTLCloudMonitoringUI.md

```diff
@@ -11,16 +11,16 @@ This document describes how to access the Monitoring UI, and provides a breakdow
 
 ![MonitoringLeftNavScreenShot](../../images/MonitoringLeftNavScreenShot121317.JPG)
 
-##### Events
+##### [Events](https://www.ctl.io/knowledge-base/cloud-application-manager/monitoring/events.md)
```

Comment:
Change this to `(../Monitoring/Events.md)`

Links in KB articles that reference other KB articles should be in the form of a filename. If the article is in the same directory, you just have to use the filename. If it is in the parent directory it would have to be prefixed with `../`. If it is in a sister directory, it should be prefixed with `../Sister/`. Also the case must match exactly or the check will fail.
There should never be any links in the article that start with `https://www.ctl.io/knowledge-base`